cedar-policy-core/src/est.rs

```diff
@@ -2284,7 +2284,42 @@ mod test {
                                                 "attr": "email"
                                             }
                                         },
-                                        "pattern": "*@amazon.com"
```

Comment:
Can we add a test demonstrating the behavior when `Literal` contains more both more than one character as well as a multibyte character?

Reply:
Added a test.